### ISSUE TITLE
`op-node/v1.10.2` Build Issue

### ISSUE THREAD

**Issue by 86b** [ethereum-optimism/optimism#13528]
`op-node/v1.10.2` doesn't seem to be building correctly.

Ubuntu 24.04
`go version go1.23.1 linux/amd64`

`make op-node`:

```
just  ./op-node/op-node
make: just: Permission denied
make: *** [Makefile:100: op-node] Error 127
```

If you troubleshoot this issue and eventually get it to build adjusting the Makefile, the version shows some outdated `0.0.0-` when running `op-node --version`

`op-node/v1.10.1` builds just fine with no troubleshooting or Makefile adjustments

**Comment by rayn316** [comment]
same error.
And an error occurred while executing `pnpm install`
` ERR_PNPM_NO_PKG_MANIFEST  No package.json found in /tmp/optimism`

**Comment by Pete-LunaNova** [comment]
We are also encountering this issue with `go version go1.22.10 linux/amd64` on Debian Bookworm.

**Comment by wangjiangw** [comment]
we also meet the same issue, the wrong message like below : 

![Image](https://github.com/user-attachments/assets/a85eaa7c-25fe-4791-9adb-d660c015d73a)

**Comment by Tommyhom** [comment]
Also have same issue as @wangjiangw. Any fix yet?

**Comment by wangjiangw** [comment]
> Also have same issue as [@wangjiangw](https://github.com/wangjiangw). Any fix yet?

@cmary6395 , no, I have tried to build successfully in a new machine which looks normally, so i think it maybe something wrong with dependency cache.

**Comment by SimonKraus** [comment]
I strongly second that the migration to `just` runner wasn't tested enough and worse: not documented at all. 

Even when installing `just` (https://mpr.makedeb.org/ is down) and fixing the permission issues I ended up with `just` not knowing what shell to use on a default ubuntu22. 

Ended up falling back to the oldschool make build. 

Please fix/improve/document this for the next release.

**Comment by sigma** [comment]
My bad. The main problem is a mishandling of multiple tags pointing to the same commit, which leads to incorrect version expansion (and a spurious "argument" making it to the build command, making it fail with the help message seen above)

**Comment by sigma** [comment]
a workaround to rebuild op-node at that version would be to skip the version detection:

```
make VERSION=v1.10.2 op-node
```

**Comment by wangjiangw** [comment]
> a workaround to rebuild op-node at that version would be to skip the version detection:
> 
> ```
> make VERSION=v1.10.2 op-node
> ```

@sigma it looks that the issue is still un-resolved ?
